find_high_boundary_areas.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy import signal
+
+# Common edge detection method for grayscale images (perfect for this)
+def find_high_boundary_areas(mask, num_targets=10):
```

Comment:
Add type annotations `(mask: np.ndarray, num_targets: int = 10) -> THE_TYPE_OF_COORDS (idk what it is)`: